cypress/support/commands.js

```diff
@@ -131,4 +132,8 @@ Cypress.Commands.add("makePOSTRequest", (requestBody) => {
       }
     ))
   )
-});
\ No newline at end of file
+});
+
+Cypress.Commands.add("injectSRAxe", () => {
```

Comment:
⚠️ ran into this issue when running cypress locally: https://github.com/component-driven/cypress-axe/issues/134
so had to add this logic to set the correct path to cypress and add this option: https://github.com/component-driven/cypress-axe/pull/150